TeaDiary/Scripts/App/constants.js

```diff
@@ -12,4 +12,16 @@ const SEARCT_TYPE = {
 const TEA_LIST_TITLE = "Your tea list:";
 const NO_TEAS_TITLE = "No teas were found :(";
 
-const TEA_ELEMENT_ID_PREFIX = "displayedTeaElementNumber";
\ No newline at end of file
+const TEA_ELEMENT = {
+    PREFIX: "displayedTeaElement",
+    ID: "Id",
+    CHECKBOX: "Checkbox",
+    LINK: "Link"
+}
+
+const MOAR = ">> More";
+
+const NUMBER = "#";
+const NAME = "Name";
+const TYPE = "Type";
+const LINK = "Details";
```

Comment:
It's a good time to start grouping your constants. You have a bunch of stuff only needed in `teaListDisplayer`. Create `TeaListDisplayer.constants.js` and put it there.